tests/parent-child-node.html

```diff
@@ -0,0 +1,87 @@
+<!DocType HTML>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script src="../node_modules/@webcomponents/webcomponents-platform/webcomponents-platform.js"></script>
+    <script src="../node_modules/@webcomponents/template/template.js"></script>
+    <script>
+      ShadyDOM = {force: true};
+    </script>
+    <!-- <script src="../shadydom.min.js"></script> -->
+
+    <script src="../node_modules/wct-browser-legacy/browser.js"></script>
+  </head>
+  <body>
+    <template>Text<!--Comment--><p>Element</p></template>
+
+    <div></div>
+
+    <script>
+      var childNodes,
```

Comment:
We should define these variables as `let` and assign them in a `setup()`. Also, all tests should be wrapped in a `suite()`

Reply:
You can see the other test suites for how that is done.